packages/blade/ios/blade/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
We need to add proper icons for the app. We already have these assets in the blade-old. Do it now or later?

Reply:
I've a task for configuring fonts for both web and RN so will do it together